### ISSUE TITLE
The Accessibility testing status section of the Accessibility tab should follow the Development considerations

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon-website#3960]
## Summary
The new information for accessibility testing is very welcome, but it could be better positioned and potentially needs its heading hierarchy addressed

## Relevant information
The new accessibility testing information appears at the top of the Accessibility tab for each component.

While this information will be valuable to some, it is arguably less valuable to designers (the primary target group for the Accessibility tab) than the previously existing information on the page (What Carbon provides, Design recommendations, Development considerations). This is especially the case since the accessibility summary has already been surfaced near the top of the Usage page. As well, activating any of that summary information on the Usage page relocates the user to this section. In other words, the content is already highly discoverable without being placed at the start of the accessibility information.

A concern with it preceding the main guidance intended for designers ("What Carbon provides") is that designers are less likely to identify the key concepts on the page, when confronted with testing information that may not be pertinent to their role.

I recommend it be placed at the bottom of the page. Its heading could be listed as part of the in-page TOC links at the top (from which it is currently absent), like so:
[What Carbon provides](https://carbondesignsystem.com/components/form/accessibility#what-carbon-provides)
[Design recommendations](https://carbondesignsystem.com/components/form/accessibility#design-recommendations)
[Development considerations](https://carbondesignsystem.com/components/form/accessibility#development-considerations)
[Accessibility testing status](https://carbondesignsystem.com/components/form/accessibility#accessibility-testing-statusobject-objectobject-object)

Finally, it is currently presented as an H3. Especially given its current position, this is a hierarchical failure, since it precedes the first H2 on the page. At the least, if left in this position, it needs to be promoted to an H2. If the testing information is relocated to the bottom of the page, one can make the case that the testing status is a child of the Development considerations, but I think it can also work as a peer.

# Conclusion from refinement 
We agree and we will move the testing section to the bottom ~~and add an H2 heading~~ [decided on Aug 6 call to leave as H3, mg]. Do this for all component pages. ~~The heading may be a part of the component so this may need to be a dev issue. ~~ [Since leaving as H3, unnecessary]

**Comment by mbgower** [comment]
Just wondering about the status of this.
I see there is agreement this should be done, but implementation has been kicked down the can for two quarters.

I believe this is a quick change, especially if this is left as an H3. I have attempt this as a POC on the accordion page with with https://github.com/carbon-design-system/carbon-website/pull/4184

**Comment by laurenmrice** [comment]
@alisonjoseph @mbgower We just discussed and confirmed this topic in design crit, we are going to leave this as an H3 and move it under "Development considerations".
